frontend/README.md

```diff
@@ -2,28 +2,77 @@
 
 A single-page app for the [OpenCraft Instance Manager (OCIM)](https://github.com/open-craft/opencraft).
 
-# Provisioning
+## Provisioning
```

Comment:
```suggestion
## Running locally
```

Reply:
Also, add a small introduction here that this should be run outside the Vagrant devstack to ease the use since it's just an independent React SPA.